### DISCUSSION TITLE
Welcome to Football_Forecaster Discussions!

### DISCUSSION THREAD

**Discussion by calvell0** [calvell0/Football_Forecaster#17] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by TSteinauer** [comment]
Areeba Could you do some research on what model we should use to train?

**Comment by TSteinauer** [comment]
Matt lets call sometime to discuss the structure of the SQL tables
